Olympus.Stratis/functions/fn_addChatCommands.sqf

```diff
@@ -0,0 +1,48 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Tyrone, JoramD
+ * Adds various chat commands.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call TAC_Olympus_fnc_addChatCommands
+ */
+
+["tac-heal", {
+    params ["_name"];
+
+    if (_name == "") then {
+        [ace_player] call ACEFUNC(medical_treatment,fullHealLocal);
+        ["ace_common_systemChatGlobal", format ["[TAC] Healed %1", name ace_player], ace_player] call CBA_fnc_targetEvent;
+    } else {
+        private _fnc_findMatch = {
+            params ["_name"];
+
+            private _matches = [];
+
+            {
+                if ([_name, name _x] call BIS_fnc_inString) then {
+                    _matches pushBack _x;
+                };
+            } forEach (call CBA_fnc_players);
+
+            if (count _matches == 1) exitWith {_matches select 0};
+            objNull
+        };
+
+        private _foundPlayer = _name call _fnc_findMatch;
```

Comment:
```suggestion
        private _foundPlayers = (call CBA_fnc_players) select {_name in (name _x)};
        // proceed to check if multiple, then tell user to specify in more detail, otherwise take the only (first) match
```